board/miyoo/main/apps/usb-hid/usb-hid.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import keyboard
+
+## usb hid codes: https://www.usb.org/sites/default/files/documents/hut1_12v2.pdf 
+hidCodeDict = {
+103:82, #up
+108:81, #down
+105:80, #left
+106:79, #right
+29:29, #b mapped to z
```

Comment:
why these letters for mapping? Can you add button hotkey e.g. Shift + F10 for right mouse click? Moreover there could be CTRL and ALT and 'c' , 'v' for e.g. copy/pasta manouvers.

Reply:
Hotkey mapping is not a good idea. I'm not sure if you can map hotkey in games/emus.

Reply:
Yeah, but I'm thinking to add SHIFT, ALT, TAB and MENU key for easier navigation in Win\Linux GUI. For hotkeys I guess ppl can use autohotkey app if needed.

Reply:
and add smth to L2/R2

Reply:
these three keys (shift ,alt, ctrl) have different (special) handling. I've tried to implement it, but for strange reasons, it could not be pressed together. Here are the findings:

codes for special keys:
LCTRL: 0x01 = 1
LSHIFT: 0x02 = 2
LALT: 0x04 = 4
LMETA: 0x08 = 8
RCTRL: 0x10 = 16
RSHIFT: 0x20 = 32
RALT: 0x40 = 64
RMETA: 0x80 = 128

to send special key need to be written in the format 
example
`write_report(chr(LCTRL)+NULL_CHAR*2+NULL_CHAR*5)`

Reply:
Ok we can leave it for now as is and investigate later, TY.

Reply:
I figured out, it is sum